content/workshops/ansible_tower/exercise3.0.adoc

```diff
@@ -0,0 +1,156 @@
+---
+title: "Exercise 3.0 - Using Ansible to Implement Security"
+workshops: ansible_tower
+workshop_weight: 30
+layout: lab
+---
+
+:license_url: http://ansible-workshop-bos.redhatgov.io/ansible-license.json
+
+:icons: font
+:imagesdir: /workshops/ansible_tower/images
+
+
+= Exercise 3.0 - Using Ansible to Implement Security
+
+In this exercise, we are going to use Ansible tower to run DISA STIG and NIST 800-53 evaluations of our environment.
```

Comment:
In this exercise, we are going to use Ansible Tower to run the DISA STIG and NIST 800-53 evaluations of our environment.